schemas/automated-liquidity-management/schema.json

```
@@ -0,0 +1,253 @@
+          "description": "The address of the supplied underlying token.",
+          "type": "string"
+        },
+        "underlying_token_index": {
+          "description": "The index of the underlying token in the smart contract, default 0.",
+          "type": "number"
+        },
+        "underlying_token_amount": {
+          "description": "The amount of underlying token supplied in this pool, decimal normalized.",
+          "type": "number"
+        },
+        "underlying_token_amount_usd": {
+          "description": "The amount of underlying tokens supplied in this pool, in USD.",
+          "type": "number"
+        },
+        "total_fees_usd": {
+          "description": "The total amount of revenue and fees paid in this pool in the given snapshot, in USD.",
+          "type": "number"
+        }
+      }
+    },
+    {
+      "label": "Events",
+      "tableName": "liquidity_yield_events",
+      "aggregation": "transaction",
+      "uniqueKey": ["transaction_hash", "log_index"],
+      "description": "All user events (ie, Deposit, Withdrawal)",
+      "properties": {
+          "timestamp": {
+              "description": "The timestamp of the transaction.",
+              "type": "number"
+          },
+          "chain_id": {
+              "description": "The standard id of the chain.",
+              "type": "number"
+          },
+          "block_number": {
+              "description": "The block number of the trade.",
+              "type": "number"
+          },
+          "log_index": {
+              "description": "The event log. For transactions that don't emit event, create arbitrary index starting from 0.",
+              "type": "number"
+          },
+          "transaction_hash": {
+              "description": "The hash of the transaction.",
+              "type": "string"
+          },
+          "user_address": {
+              "description": "The address that initiates the transaction (ie, the transaction signer).",
+              "type": "string"
+          },
+          "pool_address": {
+              "description": "The smart contract address of the pool.",
+              "type": "string"
+          },
+          "underlying_token_address": {
+              "description": "The contract address of the underlying token or deposited token.",
+              "type": "string"
+          },
+          "amount": {
+              "description": "The amount of token transacted, decimal normalized.",
+              "type": "number"
+          },
+          "amount_usd": {
+              "description": "The amount of token transacted, in USD.",
+              "type": "number"
+          },
+          "event_type": {
+              "description": "The type of event, corresponds to the action taken by the user (ie, deposit, withdrawal).",
+              "type": "string"
+          }
+      }
+  },
+  {
+    "label": "Incentive Claim Data",
+    "tableName": "misc_incentive_claim",
```

do we need to name this?
```suggestion
    "tableName": "automated_liquidity_incentive_claim",
```